.gitignore

```diff
@@ -63,3 +63,5 @@ fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
 fastlane/readme.md
+.DS_Store
```

Comment:
This is SO specific. Could you move that to your own `$HOME/.gitignore`?

Reply:
Okay, I'm on it